explore/src/main/scala/explore/config/sequence/GeneratedSequenceBody.scala

```diff
@@ -61,10 +61,11 @@ object GeneratedSequenceBody:
         obs.execution.config.map: config =>
           SequenceData(
             config,
-            Map(
-              ObserveClass.Science     -> obs.itc.science.selected.signalToNoise,
-              ObserveClass.Acquisition -> obs.itc.acquisition.selected.signalToNoise
-            )
+            Map.empty
```

Comment:
So, we're not displaying science and acquisition signal to noise values somewhere now? If Andy is OK with that for the demo, I'm fine, too. :)

Reply:
we can ask @andrewwstephens